### PR TITLE
Added NO_COMB annotation.

### PR DESCRIPTION
**Pull request by mkurc-ant**
vlog_to_model in some cases (e.g. DFF with enable signal) incorrectly
detects signals as combinational. This annotation explicitly specifies
input as non-combinational.

### CODE REVIEW

File: v2x/vlog_to_model.py [hunk 1]
```diff
@@ -172,6 +172,7 @@ def vlog_to_model(infiles, includes, top, outfile=None):
             clocks = run.list_clocks(infiles, top)
 
             for name, width, bits, iodir in ports:
+                nocomb = tmod.net_attr(name, "NO_COMB")
```
Comment (mithro) [inline v2x/vlog_to_model.py]: This needs a test.

Comment (mkurc-ant) [reply v2x/vlog_to_model.py]: Added test.

**Comment by mkurc-ant** [comment]
Added README.rst

**Comment by mithro** [comment]
DCO is missing....

**Comment by mkurc-ant** [comment]
@mithro Fixed that.

**Pull request merged**